.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+- '3.6'
```

Comment:
I know that indentation matters in a yml file.  Some of these lines starting with hyphen are indented and some aren't (line 3 isn't, but 5 is).  Does this file really work?

Reply:
yes